src/transaction/log_page_buffer.c

```diff
@@ -1999,11 +1990,97 @@ request_log_page_from_ps (LOG_PAGEID log_pageid)
         {
           _er_log_debug (ARG_FILE_LINE, "Sent request for log to Page Server. Page ID: %lld \n", log_pageid);
         }
-    }  
+    }
   // *INDENT-ON*
 #endif // SERVER_MODE
 }
 
+#if defined (SERVER_MODE)
+void logpb_assert_correctness_read_page_from_file_or_page_server (LOG_PAGEID pageid,
+    const LOG_PAGE * left_log_pgptr, const LOG_PAGE * rite_log_pgptr)
```

Comment:
Coding style for function definition: return type on the first line, the function name and first arguments in the second line.

Reply:
Done